stub_manager/src/lib/config.rs

```diff
@@ -0,0 +1,6 @@
+#TODO: figure out way to read value from ENV variable or use below defined!
```

Comment:
You can use [`std::env::var`](https://doc.rust-lang.org/std/env/fn.var.html) for this.